src/components/form/FormInput.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
```

Comment:
```suggestion
import type { Meta, StoryObj } from '@storybook/react';
```
then use this, everything will work after that without changing the whereabouts of packages